forge_tests/TestReconfigure.sol

```diff
@@ -565,4 +623,181 @@ contract TestReconfigureProject_Local is TestBaseWorkflow {
         assertEq(fundingCycle.number, 2);
         assertEq(fundingCycle.weight, _dataReconfiguration.weight);
     }
+
+    function testMixedStarts() public {
```

Comment:
Maybe we should have a test for a single of reconfiguring while fuzzing a ballot delay + start at, compared to configure (ie correct ballot status?), as we introduce changes in ballot state logic